pkg/kubectl/cmd/cmd.go

```diff
@@ -110,23 +110,26 @@ __custom_func() {
 }
 `
 	valid_resources = `Valid resource types include:
-   * pods (aka 'po')
-   * replicationcontrollers (aka 'rc')
+   * componentstatuses (aka 'cs')
    * daemonsets (aka 'ds')
-   * services (aka 'svc')
+   * deployments
```

Comment:
I'm aware of that, but I didn't want to make those conditionals. Especially that they already appear in the md files.